assets/components/content-types/event/event-listing.twig

```diff
@@ -6,15 +6,15 @@
           {% include '@atoms/picture/picture.twig' with {'variant':'news-thumb'} %}
         </div>
         <div class="list-group-teaser-content">
-          <p class="h5">Startup Acceleration Workshops</p>
+          <p class="h5 card-title event-title" itemprop="name">Startup Acceleration Workshops</p>
```

Comment:
Why not adding the `itemtype="http://schema.org/Event"` to the component ?

Reply:
yes right 903e564